hw/ip/kmac/lint/sha3.waiver

```diff
@@ -15,3 +15,12 @@ waive -rules {RESET_USE} -location {keccak_round.sv} -regexp {'rst_ni' is connec
 waive -rules {TWO_STATE_TYPE} -location {keccak_2share.sv} \
       -regexp {'index_z' is of two state} \
       -comment "index_z behaves as constant"
+
+waive -rules {ARITH_CONTEXT} -location {sha3pad.sv} -regexp {.*Bitlength of arithmetic operation.*is self-determined in this context.*} \
+      -comment "Carry issue when determning the bit length."
```

Comment:
It took me a while to understand what AscentLint was getting at here, but it's kind of reasonable. I wonder whether it would be better to define `block_addr_last` or something to be `block_addr_limit - 1` and then compare against that? AscentLint would stop moaning because we'd be assigning that to a (sized) variable: we'd not be doing any arithmetic without a context-supplied width.

If not, no worries. However, there's a missing "i" in "determining". :-)

Reply:
Fixed the spelling error. Since this PR is purposefully not fixing any RTL, I guess the lint waiver is ok.